Fashionz/index.html

```diff
@@ -0,0 +1,136 @@
+<!DOCTYPE html>
+<html>
+    <title>Fashionz</title>
+    <meta name="viewport" content="width=device-width", initial-scale="1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.3.0/css/all.min.css" integrity="sha512-SzlrxWUlpfuzQ+pcUCosxcglQRNAq/DZjVsC0lE40xsADsfeQoEypE+enwcOiGjk/bSuGGKHEyjSoQ1zVisanQ==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link rel="stylesheet" href="assets/styles.css">
+</html>
+
+<body class="body-bg-color">
+    <!----------START :: Header section--------->
+    <header>
+        <div class="fashionz-logo">
+            <div><img src="assets/images/logo.png" alt=""></div>
+        </div>
+        <div class="fashionz-menu-icon">
+            <div class="fashionz-menu-dropdown">
+                <div><a href="">My Account</a></div>
```

Comment:
1. Do we really need a div tag to be wrapped for these links ?
2. Why can we go with the ul tag and instead wrap these links with li tags ?

Reply:
Review comments resolved